docs/guides/configuration.md

```diff
@@ -893,6 +893,27 @@ Example configuration:
 
 The default model kind is `VIEW` unless overridden with the `kind` key. For more information on model kinds, refer to [model concepts page](../concepts/models/model_kinds.md).
 
+##### Identifier resolution
```

Comment:
Not sure if this is overkill, but since there has been some discussion in Slack, I thought I'd do at least a first pass and then we could chat about it.

cc @tobymao @treysp